arches/settings_utils.py

```diff
@@ -1,24 +1,67 @@
-import importlib
 import json
 import os
 import site
 import sys
+from contextlib import contextmanager
 
-from pathlib import Path
+import django
+from django.apps import apps, AppConfig
+from django.conf import global_settings, settings
+
+
+@contextmanager
+def move_to_end_of_sys_path(*paths):
+    _orig_sys_path = sys.path[:]
+    for path in paths:
+        if path in sys.path:
+            sys.path.remove(path)
+            sys.path.append(path)
+    try:
+        yield
+    finally:
+        sys.path = _orig_sys_path
+
+
+def arches_applications():
+    from arches import settings as core_settings
+
+    return [
+        mod.name for mod in arches_applications_modules(core_settings.INSTALLED_APPS)
+    ]
+
+
+def arches_applications_modules(installed_apps):
```

Comment:
nit: there's a lot of overlap for having both this and `arches_applications` as public methods. Since `arches_applications` just calls this, could this be moved to a private method and the callsites adjust to instead call `arches_applications`?

Reply:
Probably can make it private. The callsites grab the path, not just the name, so we can expose another tiny method like `arches_applications_paths` if you think that's a good direction.